### PR TITLE
LUN-282: Error unable to open url mailto

### PR DESCRIPTION
**Pull request by f1sh1918**
This pull request belongs to an issue on our [bugtracker](https://issues.tuerantuer.org/secure/RapidBoard.jspa?rapidView=27&view=detail).
You can find it there by looking for an issue with the key which is mentioned in the title of this pull request.
It starts with the keyword _LUN_.

Both fixes are only necessary for emulators to avoid errors. On real devices everything works as expected.

`tel` & `mailto` can only be tested on real ios devices. Just added handling for not supported devices like emulators
Impressum-> Click on mailaddress

### CODE REVIEW

File: src/services/url.ts [hunk 1]
```diff
@@ -5,5 +5,12 @@ export const addTrailingSlashToUrl = (url: string): string => (url.endsWith('/')
 
 export const openExternalUrl = (url: string): Promise<void> =>
   Linking.canOpenURL(url)
-    .then(() => Linking.openURL(url))
+    .then(supported => {
+      if (!supported) {
+        // emulators do not support certain schemes like mailto or tel
```
Comment (klinzo) [inline src/services/url.ts:10]:
```suggestion
        // iOS Simulators do not support certain schemes like mailto or tel
```

File: src/services/url.ts [hunk 2]
```diff
@@ -5,5 +5,12 @@ export const addTrailingSlashToUrl = (url: string): string => (url.endsWith('/')
...
+        console.log('Cannot handle url', url)
```
Comment (klinzo) [inline src/services/url.ts:11]: Our internal logging function should be used (`log` from sentry.ts)

**Review by klinzo — APPROVED** [review]

**Review by SteffiStuffel — APPROVED** [review]
I think we had this issue on a real device (on claras phone), not sure which on this was. Do you know, @klinzo ?

**Comment by klinzo** [comment]
Not sure anymore, I think the issue on claras phone was something different. Lets see if this happens again in the future

**Pull request merged**